src/components/Usage.jsx

```diff
@@ -19,17 +19,63 @@ const Usage = () => {
   const editing = { allowDeleting: true, allowEditing: true };
   const settings = { wrapMode: "Content" };
 
+  function flattenObject(ob) {
+    const result = {};
+
+    for (const i in ob) {
+      if (typeof ob[i] === "object" && !Array.isArray(ob[i])) {
+        const temp = flattenObject(ob[i]);
+        for (const j in temp) {
+          result[i + "_" + j] = temp[j];
+        }
+      } else {
+        result[i] = ob[i];
+      }
+    }
+    return result;
+  }
+
   useEffect(() => {
     // call the API to get the user instances
-    getUserUsage(localStorage.getItem("jwtToken"))
-      .then((data) => {
-        setUserUsage(data);
+    getUserInstances()
```

Comment:
Should this be getUserUsage instead of instance?

Reply:
The getUserUsage API can not work, the required information can be obtained through the getUserInstances API.